app/uk/gov/hmrc/nisp/controllers/NIRecordController.scala

```diff
@@ -65,8 +66,9 @@ trait NIRecordController extends NispFrontendController with AuthorisedForNisp w
               niSummary.spaYear))
 
               val tableStart = niSummary.recordEnd.getOrElse(niSummary.earningsIncludedUpTo.taxYear + 1)
+              val currentDate = new LocalDate()
```

Comment:
This should be extracted into the trait and be defined using the correct time zone.

The backend has a good example https://github.com/hmrc/nisp/blob/master/app/uk/gov/hmrc/nisp/services/SPResponseService.scala#L37